src/main/controllers/trezor-service.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+const trezor = require('trezor.js');
+const EventEmitter = require('events');
+const { publicToAddress } = require('ethereumjs-util');
+const EthereumTx = require('ethereumjs-tx');
+const BigNumber = require('bignumber.js');
+const { timeout, TimeoutError } = require('promise-timeout');
+const HDKey = require('hdkey');
+
+const _emitter = new EventEmitter();
+const list = new trezor.DeviceList();
+
+let currentSession = null;
+let currentDevice = null;
+const hexPrefix = '0x';
+
+const hardeningConstant = 0x80000000;
+const defaultAddress = [
+	(44 | hardeningConstant) >>> 0,
+	(60 | hardeningConstant) >>> 0,
+	(0 | hardeningConstant) >>> 0,
+	0
+];
+
+const getAddressByIdex = index => {
+	return defaultAddress.concat([+index]);
+};
+
+const addHexPrefix = val => {
```

Comment:
@tmchedlishvili I think I have seem similar function somewhere. Couldn't we reuse it if that is the case?